terraform/stacks/main/stack.tf

```diff
@@ -467,6 +467,7 @@ resource "random_password" "csb_rds_password" {
 }
 
 module "csb_broker" {
+  count             = var.stack_description != "development" ? 2 : 0
```

Comment:
@jameshochadel Is this right? Or does that create 2 copies of the broker in each non-development env?

Reply:
This would create 2 copies of it. If you're trying to create this resource conditionally, you would want to use `1`, which means: if the condition is true, then create 1 instance of this resource, otherwise 0.